core/src/test/java/org/frankframework/http/authentication/OAuthAccessTokenRequestTest.java

```diff
@@ -49,12 +50,12 @@ void testRetrieveAccessTokenWithClientCredentialsGrantInRequestParameters() thro
 		httpSender.configure();
 		httpSender.start();
 
-		AbstractOauthAuthenticator oauthAuthenticator = (AbstractOauthAuthenticator) AbstractHttpSession.AuthenticationMethod.CLIENT_CREDENTIALS_QUERY_PARAMETERS.newAuthenticator(httpSender);
-		HttpEntityEnclosingRequestBase request = oauthAuthenticator.createRequest(httpSender.getCredentials());
+		AbstractOauthAuthenticator oauthAuthenticator = (AbstractOauthAuthenticator) AbstractHttpSession.OauthAuthenticationMethod.CLIENT_CREDENTIALS_QUERY_PARAMETERS.newAuthenticator(httpSender);
+		HttpEntityEnclosingRequestBase request = oauthAuthenticator.createRequest(httpSender.getCredentials(), new ArrayList<>());
```

Comment:
Wellicht een idee om `createRequest` te overloaden zodat je een versie zonder NameValuePair list hebt, die deze (de huidige createRequest) aanroept met een lege lijst? 

Ipv `new ArrayList()` zou ik overigens `Collections.emptyList()` gebruiken

Reply:
> Ipv `new ArrayList()` zou ik overigens `Collections.emptyList()` gebruiken

In de huidige JDK versies zou ik overigens `List.of()` zonder parameters gebruiken, om een lege lijst te krijgen. 😉

Reply:
Omdat er geen antwoord was van Thomas heb ik er even naar gekeken, en het blijkt dat de aan de list parameters toegevoegd worden in de `createRequest()` implementaties. `List.of()` en `Collections.emptyList()` zijn beiden niet aan te passen.

Reply:
Ah excuus ik heb deze gemist. De reden hiervoor is zodat parameters door meerdere inheritance lagen aangepast kunnen worden. De lijst moet dus mutable zijn